scripts/continuous/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Since travis_wait doesn't work with docker well somewhy,
+# just use bell which prints something to stdout every 5 minutes.
+function bell() {
+	for i in {5..20..5}; do
+		sleep 5m
+		echo "CI: Still building (minutes=$i)"
+	done
+}
+
+bell &
+bell_pid=$!
+
+make &> build.log; RETVAL=$?; tail -c 3M build.log
```

Comment:
replace `;` with new lines may be? Just for readabillity.
Here and below

Reply:
Thanks, I separated this with new lines